ipfshttpclient/filescanner.py

```diff
@@ -512,7 +512,7 @@ def __init__(
 			#
 			# Note: `os.fwalk` support for binary paths was only added in 3.7+.
 			directory_str_or_fd = directory_str  # type: ty.Union[ty.AnyStr, int]
-			if HAVE_FWALK and (not isinstance(directory_str, bytes) or HAVE_FWALK_BYTES):  # type: ignore[unreachable]  # noqa: E501
+			if HAVE_FWALK and (not isinstance(directory_str, bytes) or HAVE_FWALK_BYTES):  # noqa: E501
```

Comment:
Same here: `noqa: E501` is “line too long” and was added mostly likely because the mypy `type: ignore` comment blew up the line length in the first place.

Reply:
Removed.